hack/generate/samples/helm/memcached.go

```diff
@@ -90,21 +94,16 @@ func (mh *MemcachedHelm) Run() {
 		"# +kubebuilder:scaffold:rules", policyRolesFragment)
 	pkg.CheckError("adding customized roles", err)
 
-	log.Infof("generating OLM bundle")
-	err = mh.ctx.DisableOLMBundleInteractiveMode()
-	pkg.CheckError("generating OLM bundle", err)
-
-	err = mh.ctx.Make("bundle", "IMG="+mh.ctx.ImageName)
-	pkg.CheckError("running make bundle", err)
-
-	err = mh.ctx.Make("bundle-build", "BUNDLE_IMG="+mh.ctx.BundleImageName)
-	pkg.CheckError("running make bundle-build", err)
+	pkg.RunOlmIntegration(mh.ctx)
```

Comment:
Since this is out of scope for this work, could you explain the motivation for this refactor?

Reply:
All samples run the same steps to integrate the project with OLM. So, instead of duplicate the code it was centralized in the utils.